clients/unshaded/src/main/java/tachyon/client/file/options/CompleteFileOptions.java

```diff
@@ -81,6 +44,15 @@ public long getUfsLength() {
   }
 
   /**
+   * @param ufsLength the UFS file length to use
+   * @return the builder
```

Comment:
Now that we aren't using builders we should update a lot of these `@return` javadocs